third_party/rust/ppoprf/v0_3/BUILD.gn

```diff
@@ -50,7 +50,7 @@ cargo_crate("lib") {
     "//third_party/rust/rand_core/v0_6:lib",
     "//third_party/rust/serde/v1:lib",
   ]
-  aliased_deps = {
-    rand_core_ristretto = "//brave/third_party/rust/rand_core/v0_5:lib__rlib"
-  }
+  # aliased_deps = {
```

Comment:
why is this commented out?

Reply:
was experimenting while resolving some build errors, forgot to remove